### PR TITLE
Smoother sliders

### PR DESCRIPTION
**Pull request by tomdenboon**
After doing a lot of research on how to solve this problem. The only solution that I found was by only updating the value if the user lets go of the slider.

### CODE REVIEW

**Comment by MarcF12** [comment]
I will test it tomorrow how good it works, but sending only on release seems not ideal. I was just thinking, can’t you start a new thread that sets the value on the server with a certain sleep time in between and that the sliders sets this public variable on the ‘main’ thread?

File: Hestia/Hestia/frontend/DevicesScreen/ActivatorScreen/TableSourceActivators.cs [hunk 1]
```diff
@@ -54,10 +54,15 @@ public override UITableViewCell GetCell(UITableView tableView, NSIndexPath index
 
                 // Set the switch's state to that of the device.
                 slider.Value = (float)act.State.RawState;
-                slider.ValueChanged += delegate (object sender, EventArgs e)
+                slider.TouchUpInside += delegate (object sender, EventArgs e)
                 {
-                    act.State = new ActivatorState<object>(slider.Value, "float");
+                    //Console.WriteLine(slider.Value);
+                    act.State.RawState = slider.Value;
```
Comment (daangroot) [inline Hestia/Hestia/frontend/DevicesScreen/ActivatorScreen/TableSourceActivators.cs]: This does not update the state on the server. The way it was done before is the correct way of doing it.

File: Hestia/Hestia/frontend/resources/Globals.cs [hunk 2]
```diff
@@ -12,6 +12,7 @@ public static class Globals
         public static String ServerName { get; set; }
         public static int Port { get; set; }
         public static String IP { get; set; }
+        public static int count { get; set; }
```
Comment (daangroot) [inline Hestia/Hestia/frontend/resources/Globals.cs]: This variable is not used, why have you added it?

File: Hestia/Hestia/frontend/DevicesScreen/ActivatorScreen/TableSourceActivators.cs [hunk 3]
```diff
@@ -54,10 +54,15 @@ public override UITableViewCell GetCell(UITableView tableView, NSIndexPath index
...
+                    act.State = act.State;
+
                 };
+
+
```
Comment (daangroot) [inline Hestia/Hestia/frontend/DevicesScreen/ActivatorScreen/TableSourceActivators.cs]: 1 white line is enough.

File: Hestia/Hestia/frontend/DevicesScreen/ActivatorScreen/TableSourceActivators.cs [hunk 4]
```diff
@@ -54,10 +54,15 @@ public override UITableViewCell GetCell(UITableView tableView, NSIndexPath index
 
                 // Set the switch's state to that of the device.
                 slider.Value = (float)act.State.RawState;
-                slider.ValueChanged += delegate (object sender, EventArgs e)
+                slider.TouchUpInside += delegate (object sender, EventArgs e)
                 {
-                    act.State = new ActivatorState<object>(slider.Value, "float");
+                    //Console.WriteLine(slider.Value);
+                    act.State.RawState = slider.Value;
+                    act.State = act.State;
```
Comment (daangroot) [inline Hestia/Hestia/frontend/DevicesScreen/ActivatorScreen/TableSourceActivators.cs]: Wut?

File: Hestia/Hestia/frontend/DevicesScreen/ActivatorScreen/TableSourceActivators.cs [hunk 5]
```diff
@@ -54,10 +54,15 @@ public override UITableViewCell GetCell(UITableView tableView, NSIndexPath index
 
                 // Set the switch's state to that of the device.
                 slider.Value = (float)act.State.RawState;
-                slider.ValueChanged += delegate (object sender, EventArgs e)
+                slider.TouchUpInside += delegate (object sender, EventArgs e)
                 {
-                    act.State = new ActivatorState<object>(slider.Value, "float");
+                    //Console.WriteLine(slider.Value);
```
Comment (daangroot) [inline Hestia/Hestia/frontend/DevicesScreen/ActivatorScreen/TableSourceActivators.cs]: Pls remove debug statements if they are no longer needed.

**Pull request closed without merging**